tests/operator_tests.py

```diff
@@ -300,6 +300,14 @@ def test_invert(self):
         np_uint_inv = ~np.arange(10, dtype=np.uint)
         self.assertListEqual(np_uint_inv.tolist(), inverted.to_list())
 
+    def test_uint_bool_binops(self):
+        # Test fix for issue #1932
+        # Adding support to binopvv to correctly handle uint and bool types
+        ak_uint = ak.arange(10, dtype=ak.uint64)
+        ak_bool = ak_uint % 2 == 0
+        self.assertTrue(np.allclose((ak_uint + ak_bool).to_ndarray(),
+                                    (ak.arange(10) + ak_bool).to_ndarray(), equal_nan=True))
```

Comment:
we really only use the `assertTrue(np.allclose` style when things return floats since there's a chance they might differ slightly

For things that return uint/int, we prefer something like
```python
self.assertListEqual((ak_uint + ak_bool).tolist(), (ak.arange(10) + ak_bool).to_list())
```